src/Athena/Views/Home/Add_Classes.cshtml

```diff
@@ -86,7 +86,17 @@
     </div>
 </div>
 
+
+
 @section scripts {
+    <script type="text/javascript">
+        $(document).ready(function () {
+            $("body").css("overflow-y", "scroll");
```

Comment:
Let's move this to a stylesheet specific to this page instead of setting it via javascript.